llvm/lib/TargetParser/AArch64TargetParser.cpp

```diff
@@ -166,6 +166,12 @@ void AArch64::ExtensionSet::enable(ArchExtKind E) {
   Touched.set(E);
   Enabled.set(E);
 
+  // FIXME: Tablegen has the Subtarget Feature FeatureRCPC_IMMO which is implied
+  // by FeatureRCPC3 and in turn implies FeatureRCPC. The proper fix is to make
+  // FeatureRCPC_IMMO an Extension but that will expose it to the command line.
+  if (E == AEK_RCPC3)
+    enable(AEK_RCPC);
+
```

Comment:
Shame this is the only exception. Would it be better to put this exception into the tablegen emitter, so that at least all the dependency info is still contained in the generated `ExtensionDependencies`?